tests/Knp/Menu/Tests/Provider/ChainProviderTest.php

```diff
@@ -6,27 +6,30 @@
 use Knp\Menu\Provider\ChainProvider;
 use Knp\Menu\Provider\MenuProviderInterface;
 use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
 
 final class ChainProviderTest extends TestCase
 {
+    use ProphecyTrait;
+
     public function testHas(): void
     {
         $innerProvider = $this->getMockBuilder(MenuProviderInterface::class)->getMock();
-        $innerProvider->expects($this->at(0))
+        $innerProvider->expects($this->exactly(1))
             ->method('has')
             ->with('first')
-            ->willReturn(true)
-        ;
-        $innerProvider->expects($this->at(1))
+            ->willReturnOnConsecutiveCalls(true);
```

Comment:
You need to use a single expectation, without exactly, and put returns inside `willReturnOnConsecutiveCalls`

Reply:
I think that still doesn't work, I am missing something

https://github.com/KnpLabs/KnpMenu/pull/352/commits/067973d087365e35f4b194b33c97093df6aed22a

I don't know if in the method expect() I should put $this->any() for the single expectation.